_includes/solver-tables.html

```diff
@@ -305,10 +305,18 @@
       <td>Comm.</td> <!-- License -->
     </tr>
 
-
-
-
-
+    <tr>
+      <th>
+        <a href="http://scip.zib.de/">SCIP</a>
+        <a href="https://github.com/SCIP-Interfaces/SCIP.jl">(.jl)</a>
+      </th>
+      <td>&#10004;</td> <!-- Linear/Quadratic -->
+      <td></td> <!-- Convex conic -->
```

Comment:
That's true, but we don't implement MPB's `AbstractConicModel` and I thought that this is meant by the table column.